extras/Build/CMake/JUCEUtils.cmake

```diff
@@ -1337,6 +1347,40 @@ function(_juce_set_plugin_target_properties shared_code_target kind)
         endif()
 
         _juce_copy_after_build(${shared_code_target} ${target_name} "${output_path}" JUCE_VST3_COPY_DIR)
+    elseif(kind STREQUAL "LV2")
+        set_target_properties(${target_name} PROPERTIES
+            BUNDLE_EXTENSION lv2
+            PREFIX ""
+            SUFFIX .lv2
+            BUNDLE TRUE
+            XCODE_ATTRIBUTE_WRAPPER_EXTENSION lv2
+            XCODE_ATTRIBUTE_LIBRARY_STYLE Bundle
+            XCODE_ATTRIBUTE_GENERATE_PKGINFO_FILE YES)
```

Comment:
Do we know if we need these? I'm not familiar with macOS packaging, but in my opinion, we're just going to be building a single folder containing the dynamic library and the .ttl files, so I don't know if we need `BUNDLE_EXTENSION`, `BUNDLE` and all the `XCODE_ATTRIBUTE` stuff.

Not only that, but if we decide we're only supporting Linux, then some of these are definitely unnecessary, since they're macOS-specific.

I think the only property we want for sure is `PREFIX ""`, since that gets rid of the `lib` in front of the library name.

Reply:
I had originally just copied all of these properties from the VST3 configuration. You're definitely right that they don't do anything now since LV2 is only enabled for Linux, but I think they will be necessary. For example, I imagine Mac will want the LV2 plugins to be signed, and I think it needs to a bundle in order to be signed.